endpoint/http/src/main/java/de/fraunhofer/iosb/ilt/faaast/service/endpoint/http/RequestHandler.java

```diff
@@ -103,7 +95,7 @@ public void close() throws IOException {
         HttpRequest httpRequest = HttpRequest.builder()
                 .path(request.getRequestURI().replaceAll("/$", ""))
                 .query(request.getQueryString())
-                .body(reader.lines().collect(Collectors.joining(System.lineSeparator())))
+                .body(ByteStreams.toByteArray(request.getInputStream()))
```

Comment:
Nice to have: not use Google Guava dependency for this small task but create custom helper class